api/v1beta1/dataplane_types.go

```diff
@@ -260,6 +260,10 @@ type ServiceOptions struct {
 	// +kubebuilder:validation:Enum=LoadBalancer;ClusterIP
 	Type corev1.ServiceType `json:"type,omitempty" protobuf:"bytes,4,opt,name=type,casttype=ServiceType"`
 
+	// Name defines the name of the ingress service.
```

Comment:
`ServiceOptions` should be agnostic as to which service they describe. I can now see that line 239 should also be fixed to not mention `ingress`.

```suggestion
	// Name defines the name of the service.
```

Reply:
Done.